docs/configuration-reference.md

```diff
@@ -137,6 +138,37 @@ key | value | description
 :--|:--|:--
 field | string | The name of the field to drop.
 
+### drop_event
+
+The `drop_event` processor will remove all events where the specified field
+exactly matches one of the listed values.
+This can be used to filter datasets from ingress to service based on which
+service or namespace is used.
+
+Events that do not have a key named `field` present will be kept.
+
+**Options:**
+key | value | description
+:--|:--|:--
+field | string | The name of the field to look into
+values | list  | A list of string values to match with
+
+### keep_event
+
+The `keep_event` processor will remove all events NOT matching one of the
+specified values. Careless configuration of this filter will drop all events.
+
+It is effectively the inverse of `drop_event`.
+
+Events that do not have a key named `field` present will be kept to avoid
```

Comment:
Again, I suspect you mean events that lack a field bearing the name nominated in the processor configuration's "field" field.